build.gradle

```diff
@@ -118,16 +116,22 @@ subprojects {
     }
 }
 
-tasks.create(name: "releaseTarGz", dependsOn: configurations.archives.artifacts, type: Tar) {
-    into "${rootProject.name}-${archiveVersion.get()}"
+distributions {
+    main {
+        contents {
+            from(project(":core").jar)
+            from(project(":core").configurations.runtimeClasspath)
+            from(project(":storage:core").jar)
+            from(project(":storage:core").configurations.runtimeClasspath)
+            from(project(":storage:s3").jar)
+            from(project(":storage:s3").configurations.runtimeClasspath)
+            from(project(":storage:filesystem").jar)
+            from(project(":storage:filesystem").configurations.runtimeClasspath)
+        }
+    }
+}
+
```

Comment:
Checked, this seems to produce identical content with the previous `releaseTarGz`.

Reply:
Exactly, did the same while was refactoring.